sdk/elasticsan/Azure.ResourceManager.ElasticSan/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+This package follows the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html) which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
```

Comment:
It seems that we missed [this part](https://github.com/Azure/azure-sdk-for-net/blame/main/sdk/iotcentral/Azure.ResourceManager.IotCentral/CHANGELOG.md#L5-L9) in the template. Let me add it in the template.